lib/internal/Magento/Framework/Filesystem/Io/Sftp.php

```diff
@@ -17,7 +19,7 @@ class Sftp extends AbstractIo
     const SSH2_PORT = 22;
 
     /**
-     * @var \phpseclib\Net\SFTP
+     * @var \phpseclib3\Net\SFTP
```

Comment:
Can we use import instead of FQCN?

Reply:
The class name is also `SFTP` and I couldn't fix on an appropriate alias. Leaving it as FQCN for now seemed appropriate. Do you have any alias suggestions?

Reply:
@Den4ik, any thoughts on this one?

Reply:
I think it's better to keep it as is for now